sublime_info/__init__.py

```diff
@@ -1,6 +1,6 @@
 # Load in local modules for extension
-from sublime_info import SublimeInfo
-from errors import *
+from sublime_info.SublimeInfo import SublimeInfo
```

Comment:
Can we add a `from __future__ import absolute_import`? I think this might break Python 2 =/